.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--
+Follow the steps to create the PR:
+
+Subject: "feat/fix/docs(#issue_id): <PR Subject>"
+Assignees: "One Platform Developers"
+-->
+
+# Closes/Fixes/Resolves
+
+### <!-- Any of the keywords: Closes/Fixes/Resolves followed by #issue_id (should be separated by a space only) -->
+
+# Explain the feature/fix
+
+<!-- Please include a summary of the change and which issue is fixed. -->
+
+## Does this PR introduce a breaking change
+
+<!-- Yes/No -->
+
+<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->
+
+## Screenshots
+
+<!-- If applicable, add screenshots/gif to help explain your problem. -->
+
+<details>
+<summary>View Screenshots</summary>
+
+<!-- Add your screenshots below this line -->
+
+</details>
+
+### Ready-for-merge Checklist
+
+- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
+- [ ] Does the change have appropriate unit tests?
+- [ ] Did tests pass?
+- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
+- [ ] My changes generate no new warnings
+- [ ] I have performed a self-review of my own code
+- [ ] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
+#### If it is a new component
+- [ ] Does your component follow One Platform style guidelines?
+- [ ] Is your component responsive (supports standard - Mobile, Tab, Desktop)
+#### Browsers you have tested in
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Edge
```

Comment:
there should be Safari as well as IE11

Reply:
We can add Safari but I’m not sure we can test it in IE11

<img width="1263" alt="Screenshot 2020-07-20 at 11 37 02 AM" src="https://user-images.githubusercontent.com/12994292/87905054-568c9d00-ca7d-11ea-9645-2b64043ad32f.png">